perf/lua/benchmark.lua

```diff
@@ -132,7 +132,13 @@ end
 
 local function load_average()
     local path = '/proc/loadavg'
-    local fh = assert(fio.open(path, {'O_RDONLY'}))
+    local fh, errmsg = fio.open(path, {'O_RDONLY'})
+    if not fh then
+        -- It is a different file layout on macOS, so there is no
+        -- </proc>. Just do not initialize that field.
+        io.stderr:write(("Can't open file '%s': %s\n"):format(path, errmsg))
+        return nil
```

Comment:
`load_average()` returns a table.

Reply:
Yes, in case of succes. I suppose the absent field is better than the "incorrect" (empty table) one.